spec/cancan/ability_spec.rb

```diff
@@ -361,6 +361,20 @@ class A; include B; end
     expect(@ability.can?(:read, A)).to be(true)
   end
 
+  it "allows to check ability in a decorated class" do
+    class ADecorator < SimpleDelegator
+      def kind_of?(klass)
```

Comment:
I wonder if we can poke at Draper, change the way it is doing this instead. We could push a pull request over there just as easily.